BlueDBOnDisk/src/main/java/org/bluedb/disk/ReadWriteDbOnDisk.java

```diff
@@ -172,6 +172,7 @@ protected static void assertExistingCollectionIsType(ReadableCollectionOnDisk<?>
 	protected List<ReadWriteCollectionOnDisk<?>> getAllCollectionsFromDisk() throws BlueDbException {
 		List<File> subfolders = FileUtils.getSubFolders(path.toFile());
 		List<ReadWriteCollectionOnDisk<?>> collections = Blutils.map(subfolders, (folder) -> getUntypedCollectionForBackup(folder.getName()));
+		collections.removeIf(readWriteCollectionOnDisk -> readWriteCollectionOnDisk == null);

```

Comment:
Blutils.map can't return null and it can't return a list that contains null. So this line seems unnecessary.